src/Mobs/Behaviors/BehaviorCoward.cpp

```diff
@@ -17,7 +17,7 @@ cTraitCoward::cTraitCoward(cMonster * a_Parent) :
 
 
 
-bool cTraitCoward::ActiveTick()
+bool cBehaviorCoward::ActiveTick()
 {
 	if (m_Attacker == nullptr)
 	{
```

Comment:
@LogicParrot 
If a chicken is attack,   it's  `MoveToPosition` will be call, and  `pathfinder`  will be trigger  to move to the destination.  
(if i am wrong, please notice me that)
But  how does `pathfinder`  know the chicken need to run , or fly (jump) or not, and how fast the chicken runs to the destination?